string app_name
boolean keep_running
---
string success
